### DISCUSSION TITLE
example of api, setting features

### DISCUSSION THREAD

**Discussion by nadia-am** [CoderLine/alphaTab#619] [Q&A] [answered]
I want to add "at-track-controls" to my project as example in "https://alphatab.net/", where can i find a examples?

**Comment by Danielku15** [comment, marked as answer]
The tutorial here shows how to create a control similar like shown on the website: https://alphatab.net/docs/tutorials
Please note that alphaTab does not ship all controls out-of-the-box. It is the developers task to build a UI around alphaTab.

**Reply by nadia-am** [reply]
i saw that example and that was great but i need some more example of others feature.
tnx anyway
